### DISCUSSION TITLE
Práctica Intro Caso

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#34] [Q&A]
Hola, disculpen, en  la descripción serían:

1. El cliente (los abogados)
2. El usuario
3. El administrador

Es que me confunde porque yo al principio solo vi.
 
1. Al administrador que serian los abogados
2. Cliente que es el usuario

Pero como vi la palabra cliente y usuario me confundí.

**Comment by AlexAntonioG** [comment]
Al menos desde donde entiendo, solo serían los abogados y el usuario, debido a que estos son los que revisarían las peticiones,  a no ser que haya un encargado más que lo haga (_Administrador_).

Por el momento lo tomaré como dos. De todas formas te recomiendo que vayas como tu lo creas, el lunes en live supongo que las dudas serán contestadas c:

**Reply by GloriaCoralCerecedo** [reply]
Si exacto entendí lo mismo que tu entendí,  porque para que tendría que llenar un formulario, realizar el pago y crear la cuenta el abogado xd. Si el lunes diré mis dudas, gracias.

**Comment by MiguelGuerrero07** [comment]
Hola.

Te refieres al rellenado del Documento Word?

Si , es así , te puede ayudar la siguiente forma de llenado que yo realice.
![image](https://user-images.githubusercontent.com/69058719/154827665-311296b1-4921-4fcd-b2d0-59b4e4e9d9e0.png)

**Reply by GloriaCoralCerecedo** [reply]
No me refería a eso en si, pero muchas gracias :D

**Comment by doguedogue** [comment]
Casi siempre es lo mismo cliente que usuario.

Cliente --> Persona que va a demandar y también es usuario de la web que se está creando.
Administrador --> Abogado que va a realizar la demanda (yyyyy también es usuario pero con otro rol).

😁 espero no confundirte
